requirements/base.txt

```diff
@@ -6,9 +6,9 @@ opencv-python>=4.5.3.56
 pandas>=1.1.0
 pytorch-lightning[extra]>=1.6.0
 timm==0.5.4
-torchmetrics>=0.9.0
-torchvision>=0.9.1
-torchtext>=0.9.1
+torchmetrics>=0.9.0,<=0.9.1
```

Comment:
do we need to restrict `torchmetrics`? It's a different library by the Lightning team,